test/fuzz/transformation_inline_function_test.cpp

```diff
@@ -544,8 +546,10 @@ TEST(TransformationInlineFunctionTest, ApplyToMultipleFunctions) {
                                                      {64, 76},
                                                      {65, 77},
                                                      {66, 78}});
-  ASSERT_FALSE(
-      transformation.IsApplicable(context.get(), transformation_context));
+  ASSERT_DEATH(
+      transformation.IsApplicable(context.get(), transformation_context),
+      "Bad attempt to query whether overflow ids are available.");
```

Comment:
Sorry for the confusion but there seem to be no assertions in the `IsApplicable`. Why do we need an `#ifndef` here?

Reply:
This is because we use NullOverflowIdSource in TransformationContext.  It doesn't even let you ask if there are overflow ids:

```
  bool HasOverflowIds() const override {
    assert(false && "Bad attempt to query whether overflow ids are available.");
    return false;
  }
```

The idea is that during fuzzing sufficient ids should be provided so that this question should never be asked.

But it does mean that one needs death tests (or to pass in a different overflow id source - one that always says no but that doesn't assert) for the tests.

Reply:
Ok, that makes sense. Thanks.